demodata/setup-topology/setup-topology-aws.sh

```diff
@@ -0,0 +1,30 @@
+bin/pulsar-admin --admin-url "$PULSAR_ADMIN_URL" topics list tenant-1/namespace-1
+if [ $? -eq 0 ]; then
+  echo "Topics found. Skipping topology setup."
+else
+
+  echo "Setting up topology."
+  x=0
+
+  for i in {1..10}; do
+
+    echo "Creating tenant-$i"
+    bin/pulsar-admin --admin-url "$PULSAR_ADMIN_URL" tenants create "tenant-$i"
+
+    for j in {1..20}; do
+
+      echo "Creating tenant-$i/namespace-$j"
+      bin/pulsar-admin --admin-url "$PULSAR_ADMIN_URL" namespaces create "tenant-$i/namespace-$j"
```

Comment:
Your changes result in namespaces with the same name in different tenants - I wasn't sure if that is ok, that's why I wrote that weird math equations : D What do you think?

Reply:
Yeah, I just simplified it because I was not sure which numbers I have to increase to get the full 5000 topics. We can change it back if you want, I was just too lazy too get into the math equation 😄

Reply:
Maybe you could just add another y variable that is summed up :D that is so much easier, I don't know why I didn't think of that ^^